internal/provider/loadbalancer/resource_loadbalancer.go

```diff
@@ -49,15 +50,58 @@ func resourceLoadBalancerCreate(ctx context.Context, data *schema.ResourceData,
 	if err != nil {
 		return diag.FromErr(err)
 	}
-	lb, err := c.HarvesterLoadbalancerClient.
+
+	_, err = c.HarvesterLoadbalancerClient.
 		LoadbalancerV1beta1().
 		LoadBalancers(namespace).
 		Create(ctx, toCreate.(*loadbalancerv1.LoadBalancer), metav1.CreateOptions{})
 	if err != nil {
 		return diag.FromErr(err)
 	}
+
 	data.SetId(helper.BuildID(namespace, name))
-	return diag.FromErr(resourceLoadBalancerImport(data, lb))
+
+	loadbalancer, err := resourceLoadBalancerWaitIPAddress(ctx, meta, name, namespace)
```

Comment:
The `if err != nil` below will make the create operation fail if the load balancer doesn't obtain an IP within the time limit. Note that the time limit for obtaining IP addresses won't respect the user-configurable timeouts.

Also note that if the create fails due to not getting an IP in time, the load balancer resource won't be imported into the Terraform state, since that only happens at the very end of this method.

It may be better to catch the error `"no address was populated for the loadbalancer"`, log it on the console and continue.